### ISSUE TITLE
Button ID is undefined

### ISSUE THREAD

**Issue by DesignyourCode** [niuware/mui-rte#67]
Hi. Awesome package, this is working really nicely in my project.

I have come across a minor thing, in which I see some console warnings where id is coming out as undefined for each button.

```
[DOM] Found 9 elements with non-unique id #undefined: (More info: https://goo.gl/9p2vKq)
<button class=​"MuiButtonBase-root MuiIconButton-root" tabindex=​"0" type=​"button" id=​"undefined" aria-label=​"Bold">​…​</button>​
<button class=​"MuiButtonBase-root MuiIconButton-root" tabindex=​"0" type=​"button" id=​"undefined" aria-label=​"Italic">​…​</button>
...
```

**Comment by niuware** [comment]
@DesignyourCode Thank you, I will check it.

**Comment by sandeepone** [comment]
I think this will fix ``` id={style.id || style.name + 'Id'} ```
https://github.com/niuware/mui-rte/blob/00f2e6548ff1cbe35b4d193903a2b33f187f85ec/src/components/Toolbar.tsx#L251

**Comment by DesignyourCode** [comment]
Is that something you need to add into the package? Or something I can add to my code?

**Comment by sandeepone** [comment]
@DesignyourCode add into the package

**Comment by niuware** [comment]
This is fixed on version 1.11.0
